src/desktop/apps/artwork/components/show_artworks/query.coffee

```diff
@@ -5,7 +5,10 @@ module.exports = """
         name
         href
         type
-        artworks(all: true, size: 50, exclude: [$id]) {
+        counts {
+          eligible_artworks
+        }
+        artworks(size: 20, exclude: [$id]) {
```

Comment:
Just out of curiosity, why is this being repeated here?

Reply:
We use two different query fragments for artworks in a fair show vs. non-fair show (both suffered from the `all: true`). Might be room for consolidation but out-of-scope of this PR 😅

Reply:
Oh I definitely didn’t mean to do any of the sorts as part of this PR 👍 , was just wondering.